symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/model/ActivityInfo.java

```diff
@@ -14,7 +14,7 @@
  */
 @Getter
 @Setter
-@Accessors(fluent = true)
+@AllArgsConstructor
```

Comment:
that's is an API breaking change, the class is marked as experimental but still are we sure about it?

Reply:
because we cannot use the object in freemarker template with the fluent accessors, that's why I removed it. But if we decide not to use template at all, we can revert this change.

Reply:
I'm ok to modify it as it was marked as experimental